.circleci/config.yml

```diff
@@ -643,16 +643,7 @@ jobs:
       - DEPLOY_DOWNTIME: 1800 #30M
     steps: *deploy_steps
 
-  upload_packages_osx:
-    <<: *infrastructure_config
-    steps:
-      - checkout
-      - attach_workspace:
-          at: *packages_workspace
-      - *setup_secrets
-      - *upload_github_release_asset
-
-  upload_packages_linux:
+  upload_release_packages:
```

Comment:
How does `upload_release_artifacts` work with one `aws s3 cp --acl public-read ${PACKAGE_TARBALL:?} s3://${S3_RELEASES_BUCKET:?}/aeternity-latest-${PKG_SUFFIX:?}.tar.gz`? It feels like it uploads only one package to S3? Is that on purpose?

Reply:
Good catch!

Reply:
fixed